front/src/routes.js

```diff
@@ -25,6 +26,9 @@ const Routes = () => {
         <Stack.Screen
           name='Register'
           component={RegisterScreen} />
+        <Stack.Screen
```

Comment:
I create in another PR a home screen too, but i adjust it. But i think that a home screen is unnecessary too, because in login screen you can go to register screen